airflow-core/src/airflow/ui/src/pages/TaskInstance/Logs/TaskLogContent.tsx

```diff
@@ -31,8 +34,50 @@ type Props = {
   readonly wrap: boolean;
 };
 
+const ScrollToButton = ({
+  direction,
+  onClick,
+}: {
+  readonly direction: "bottom" | "top";
+  readonly onClick: () => void;
+}) => {
+  const { t: translate } = useTranslation("common");
+
+  return (
+    <Tooltip
+      closeDelay={100}
+      content={translate("scroll.tooltip", {
+        direction: translate(`scroll.direction.${direction}`),
+        hotkey: direction === "bottom" ? "↓" : "↑",
```

Comment:
the hotkey text is missing the modifier key (ctrl on linux but the other apple key on mac?)